packages/orbit-components/src/InputSelect/InputSelect.ct-story.tsx

```diff
@@ -68,6 +68,12 @@ export function InputSelectStory() {
         placeholder="Custom Placeholder"
         disabled
       />
+      <InputSelect
+        options={simpleOptions}
+        label="Custom label (readOnly)"
+        placeholder="Custom Placeholder (readOnly)"
```

Comment:
Instead of a placeholder, please add a `defaultSelected` so we can assert the correct logic of rendering the `X` button